### ISSUE TITLE
Encoding issues?

### ISSUE THREAD

**Issue by ghost** [delight-im/FreeGeoDB#4]
See https://news.ycombinator.com/item?id=10277248

**Comment by ocram** [comment]
Thanks! That's definitely a problem that has to be fixed quickly.

The current encoding errors were a direct result of the build process outlined here: https://github.com/delight-im/FreeGeoDB/blob/f08c26cccf3f3f751fc5d1d0d3abae878d7b968c/Source%20and%20Build%20Instructions.md#build-instructions

So either the build process is wrong, some mistake had been done during that process, or the source material itself has the encoding problems.

Can someone else check the Natural Earth data perhaps? The cities ("populated places") seem to be a good place to start.

**Comment by ocram** [comment]
It was indeed the source data that was wrong!

Some examples from the discussions here and elsewhere were the following ones:
- Ålesund => Plesund
- Bodø => Bodi
- Tønsberg => Tdnsberg
- Zürich => Zdrich
- La Réunion => La Rcunion
- Rhône-Alpes => RhAne-Alpes

We opened the original shapefiles from Natural Earth with QGIS to check again -- and the encoding errors were there as well.

This has already been reported to Natural Earth, it seems, but not yet been fixed:
- http://www.naturalearthdata.com/forums/topic/bad-adm1name-encoding-in-version-3-0-0-and-missing-diacritics-in-name/
- http://www.naturalearthdata.com/forums/topic/unicode-encoding-issue-ne_10m_lakes-dbf/

Fortunately, it was possible to recover the correct data from an older version of the "cities" set that was not damaged.

The JSON and CSV versions of the "cities" set have been fixed already: https://github.com/delight-im/FreeGeoDB/commit/689973a35d294ee272b40398f05ec9709ff0d545
